client/new-stack-editor/lib/index.coffee

```diff
@@ -36,27 +43,25 @@ module.exports = class StackEditorAppController extends AppController
     @stackEditor.on 'InitializeRequested', @bound 'initializeStack'
 
 
-  openEditor: (templateId) ->
+  openEditor: (templateId, reset = no) ->
 
     @fetchStackTemplate templateId, (err, template) =>
       return showErrorNotification err  if err
-      @stackEditor.setTemplateData template
+      @stackEditor.setTemplateData template, reset
 
     markAsLoaded templateId
 
 
   openStackWizard: (handleRoute = yes) ->
 
-    console.trace()
-    log '::openStackWizard', handleRoute
-
-    markAsLoaded null
+    new StackWizardModal { handleRoute }
+    markAsLoaded()
 
 
-  reloadEditor: (templateId, skipDataUpdate) ->
+  reloadEditor: (templateId) ->
 
-    console.trace()
-    log '::reloadEditor', templateId, skipDataUpdate
+    delete @templates[templateId]
+    @openEditor templateId, reset = yes
```

Comment:
aha, you already felt the same but applied another solution 🤔 
i liked it, but i am still in favor of using an object, to be less error prone.